src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -35,6 +36,13 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig)(implicit v
       JacksonFactory.getDefaultInstance, getDataProcServiceAccountCredential)
       .setApplicationName("dataproc").build()
   }
+
+  private lazy val compute = {
+    new Compute.Builder(GoogleNetHttpTransport.newTrustedTransport,
+      JacksonFactory.getDefaultInstance, getDataProcServiceAccountCredential)
+      .setApplicationName("firecloud:leonardo").build()
```

Comment:
I think @asingh7115 is making the appName configurable in her PR; I'll rebase in those changes.